webapp/src/components/Analysis/UtterancesTable.tsx

```diff
@@ -390,6 +399,41 @@ const UtterancesTable: React.FC<Props> = ({
   ];
 
   const searchString = constructSearchString(pipeline);
+
+  const importProposedActions = (file: File) => {
+    const fileReader = new FileReader();
+    fileReader.onload = ({ target }) => {
+      if (target) {
+        const result = target.result as string;
+        const [header, ...rows] = result.split(/\r?\n/).slice(0, -1);
+        if (rows.length === 0) {
+          raiseErrorToast("There are no records in the CSV file.");
+          return null;
+        }
+        const headers: string[] = header.split(",");
+        if (
+          ![config.columns.persistent_id, "proposed_action"].every(
+            (h, index) => headers.includes(h) || headers.indexOf(h) === index
+          )
+        ) {
```

Comment:
This makes no sense. I guess you meant `&&` instead of `||`, otherwise this will still pass if the headers are out of order, which is the problem I raised [here](https://github.com/ServiceNow/azimuth/pull/465#discussion_r1119560156). In that case, I think `headers.includes(h)` is useless.
Why not simplify to this?
```suggestion
        if (headers !== `${config.columns.persistent_id},proposed_action`) {
```

Reply:
Ohh yeah, this approach is so simple. I couldn't think of it this way earlier. thank you :)